packages/web-client/app/components/card-pay/deposit-workflow/transaction-setup/index.ts

```diff
@@ -19,18 +20,19 @@ interface token {
   description: string;
   icon: string;
 }
-const TOKENS: token[] = [
-  {
-    symbol: 'DAI',
-    description: 'USD-based stablecoin',
-    icon: 'dai-token',
-  },
-  {
-    symbol: 'CARD',
-    description: 'ERC-20 Cardstack token',
-    icon: 'card-token',
-  },
-];
+
+const DAI_TOKEN = {
```

Comment:
It could be nice to build on the `Token` type in `app/utils/web3-strategies/types.ts` for this.

Reply:
Yep, was thinking of doing that as a follow-up PR.